packages/react-text/src/components/Headline/Headline.tsx

```diff
@@ -17,7 +17,7 @@ export const headlineClassNames: SlotClassNames<TextSlots> = {
  * Styles for the root slot
  */
 const useStyles = makeStyles({
-  root: typographyStyles.headline,
+  root: typographyStyles.subtitle1,
```

Comment:
Is this a new Fluent typography naming? If yes, we'll need to go ahead and rename the components too for coherence, if not then these should have the same name as the component.

Reply:
The global design tokens figma and the text figma lists the same typography styles, but react-text component doesn't seem to match. I'm not sure where the names of these components were listed in the figma. I think that should be a follow on PR.

Reply:
Definitely in a follow-up PR. Just wanted to check if we had changed the naming given that, as of the time of development of the components, the names matched the component names. I'll look into it.

Reply:
FYI, created issue #22700 for tracking